app/assets/stylesheets/styles/components/crops/components/_crop_info_container.scss

```diff
@@ -31,6 +43,21 @@
   }
 }
 
+.small-crop img {
+  /*-webkit-filter: grayscale(50%);
+  filter: grayscale(50%);*/
+  @include filter(grayscale(50%));
```

Comment:
I will! I need to test the changes on staging first, then I'll make the changes suggested by Hound.